LINKS.md

```diff
@@ -136,11 +137,14 @@
 * [Remove.bg](https://www.remove.bg/) - Remove fundos de imagens automaticamente
 * [Figma](https://www.figma.com/) - Desenhe projetos online de apps, softwares e websites
 * [Pixilart](https://www.pixilart.com/draw) - Desenhe pixel arts online
+* [Whimsical](https://whimsical.com/KPBSvhYr6TP7z3EkifffUi) - Comunique-se visualmente na velocidade do pensamento - fluxogramas colaborativos, wireframes, notas adesivas e mapas mentais
+* [Mockup](https://mockup.io/about/) - Visualize e colabore no design de aplicativos para dispositivos móveis
```

Comment:
Adicione novos links no final da lista da categoria e altere a página para o início do site.

Reply:
correção aplicada